open-machine-learning-jupyter-book/assignments/data-science/estimation-of-COVID-19-pandemic.ipynb

```diff
@@ -13,10 +13,25 @@
   },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    df['Rt'] = df['ninfected'].rolling(8).apply(______ x: x[4:].sum()/x[:4].sum())
</pre><p>def get_rt(df)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/open-academy/machine-learning/pull/156/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='37'/>

Reply:
<p>Finished it</p>